examples/double_gyre.jl

```diff
@@ -165,15 +169,15 @@ run!(simulation)
 
 # # A neat movie
 
-# We open the JLD2 file, and extract the `grid` and the iterations we ended up saving at,
+# We open the JLD2 file, and extract the `grid` and the iterations we ended up saving at.
 
 using JLD2
 
 filename = "double_gyre.jld2"
 
-u_timeseries = FieldTimeSeries(filename, "u"; grid = grid)
-v_timeseries = FieldTimeSeries(filename, "v"; grid = grid)
-s_timeseries = FieldTimeSeries(filename, "speed"; grid = grid)
+u_timeseries = FieldTimeSeries(filename, "u"; architecture = CPU())
+v_timeseries = FieldTimeSeries(filename, "v"; architecture = CPU())
+s_timeseries = FieldTimeSeries(filename, "speed"; architecture = CPU())
```

Comment:
CPU is default and pretty much always desired (unless you want to do post-processing on the GPU) so we may not need to specify `architecture` here